orchestrator-salt/src/main/resources/salt/salt/datalake_resize/check_atlas_updated.sls

```diff
@@ -0,0 +1,8 @@
+include:
+  - datalake_resize.check_atlas_updated
+
+check_atlas_updated:
+  cmd.run:
+    - name: /opt/salt/scripts/check_atlas_updated.sh {{salt['pillar.get']('check_atlas_updated:max_retries')}}
```

Comment:
if the intention is:
```
Objective of this is to enable customers to run the appropriate salt commands to verify whether Atlas is up to date before initiating the resize process. 
```
then why is it included in the top? this would mean this is running always on cm server instance

Reply:
ok, I might remember wrong or I've confused the state with the pillar one, but now it seems ok to me